FakeItEasy.sln

```diff
@@ -199,6 +204,7 @@ Global
 		{DC5EAE30-96E3-4D38-B3C5-33E587C9FB76} = {AD790E7D-D664-4B5F-8BE8-01507A2A9F3B}
 		{5D41D463-D53B-4ADC-90AD-44AA03D6340D} = {AD790E7D-D664-4B5F-8BE8-01507A2A9F3B}
 		{575A3C3B-17BA-4A8A-9F53-05320913D0A6} = {AD790E7D-D664-4B5F-8BE8-01507A2A9F3B}
+		{AD790E7D-D664-4B5F-8BE8-01507A2A9F3B} = {59466C4D-C407-4DD9-B77B-D99162B9FC7C}

```

Comment:
Unless I read this wrong, it means that `tests` has moved under `src`...

Reply:
You are correct. And I have _no idea_ how I managed that.